Makefile

```diff
@@ -9,14 +19,20 @@ aws-deps:
 azure-deps: jq-dep
 	npm install -g azure-cli
 
-jq-dep:
+jq-dep: $(GOPATH)/bin/jq
+
+$(GOPATH)/bin/jq:
 	@echo "Downloading jq..."
 	mkdir -p $(GOPATH)/bin
 	wget "https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64" -O $(GOPATH)/bin/jq
 	chmod "+x" $(GOPATH)/bin/jq
```

Comment:
jq is Go code ? It seems a little odd to put it on the GOPATH if it is not Go code that will be built and placed there. This bootstrap may even fail if the binary downloaded fails to run on the host linux for some reason. I understand the good motivation on helping the user to bootstrap deps... but I gotta a strange feeling about this...but perhaps it is just a stupi feeling :-)